docs/workloads/common/data/pysparkle/pysparkle_quickstart.md

```diff
@@ -13,32 +13,26 @@ keywords:
-built to streamline data processing workflows with Apache Spark.
+[PySparkle](https://github.com/amido/stacks-azure-data/tree/main/pysparkle) is a Python library built to streamline
+data processing workflows with Apache Spark.
 
-As an integral component of the Stacks Data Platform, PySparkle is our example of data transformation
-in action. It's designed to showcase the capabilities of our platform. The application operates
-on [The Movies Dataset](https://www.kaggle.com/datasets/rounakbanik/the-movies-dataset) available
-on Kaggle, which contains metadata and ratings for 45,000 movies released on or before July 2017.
-
-**The Movies Dataset** contains a number of CSV files and includes values of different types,
-including nested datatypes and arrays, which give us enough complexity for transformations.
+As an integral component of the Stacks Data Platform, PySparkle is our example of data transformation in action.
+It's designed to showcase the capabilities of our platform.
 
 ## Key components
 
-The naming convention for the data pipelines originates in Databricks' Medallion Architecture, which
-emphasises structured data transformation layers. The key components of PySparkle include:
+The naming convention for the data pipelines originates in Databricks' Medallion Architecture, which emphasises
+structured data transformation layers. The key components of PySparkle include:
 
-- **Silver processing**: This represents the 'Bronze to Silver' layer transformations. It accepts
-raw data (CSV files) as an input and converts it into a more structured delta format.
-- **Gold processing**: This embodies the 'Silver to Gold' layer transformations. It performs
-advanced transformations, creating optimised, aggregated data sets for downstream consumption.
-- **Data Quality**: PySparkle performs data quality checks according to user-provided
-configurations, ensuring the reliability and accuracy of the data.
+- **Silver processing**: This represents the 'Bronze to Silver' layer transformations. It accepts raw data of various
+formats (e.g. CSV, Parquet, JSON, Delta) as an input and converts it into a more structured Delta format.
+- **Gold processing**: This embodies the 'Silver to Gold' layer transformations. It performs advanced transformations,
+creating optimised, aggregated data sets for downstream consumption.
+- **Data Quality**: PySparkle performs data quality checks according to user-provided configurations, ensuring the
+reliability and accuracy of the data.
 
-We have chosen Azure Data Lake Storage (ADLS Gen2) as both the source and destination systems for
-our data processing due to its scalability, security features, and seamless integration with our
-data processing workflows.
+We have chosen Azure Data Lake Storage (ADLS Gen2) as both the source and destination systems for our data processing
+due to its scalability, security features, and seamless integration with our data processing workflows.
```

Comment:
This is repeated elsewhere so think we can remove here.

Reply:
done